test/v1/mocks/services/MockSubmitReturnRequestService.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
```

Comment:
Please refactor file and names to MockSubmitReturnService for consistency instead of MockSubmitReturn**Request**Service